src/TheThingsNode.cpp

```diff
@@ -293,16 +326,23 @@ void TheThingsNode::showStatus()
 void TheThingsNode::configInterval(bool enabled, uint32_t ms)
 {
   this->intervalMs = ms;
```

Comment:
Shouldn't `pttn` be cleared here as well? Like L341

Reply:
Nope because it's done just after by the call to 
` configInterval(enabled);`
but I admit it's not read friendly, so I changed this for consistency